Makefile

```diff
@@ -311,28 +311,24 @@ docker-test: ##@tests Run tests in a docker container with golang.
 
 test: test-unit ##@tests Run basic, short tests during development
 
-test-unit: SHELL := /bin/bash -o pipefail
-test-unit: UNIT_TEST_PACKAGES = $(shell go list ./...  | \
+test-unit: export BUILD_TAGS ?= BUILD_TAGS
+test-unit: export GOTEST_EXTRAFLAGS ?= GOTEST_EXTRAFLAGS
```

Comment:
This seems pretty redundant.

Reply:
I can remove `BUILD_TAGS`, because it is not set in `Makefile` currently.
As for `GOTEST_EXTRAFLAGS` - it is needed to passthrough the env variable, when set in `Makefile`, for example here:
https://github.com/status-im/status-go/blob/81b94b7a4e7ed81f6d0ae78cf3dcbda950e5d278/Makefile#L328-L329
Or do you want to pass it as an argument to the script?

Reply:
I think the line 51 is pointless. And that variable should be set per-target using `?=`.